element_animal/subject.py

```diff
@@ -158,11 +157,13 @@ class SubjectDeath(dj.Manual):
 
 
 @schema
-class SubjectCullMethod(dj.Manual):
+class SubjectCull(dj.Manual):
     definition = """
     -> Subject
     ---
-    cull_method:    varchar(255)
+    cull_method='': varchar(255)
+    cull_reason='': varchar(255)
+    cull_notes='': varchar(1000)
```

Comment:
Does it make sense to keep this to n^2 -1 for memory management? I've seen that done elsewhere, but wasn't sure about the motivation
```suggestion
    cull_notes='': varchar(1023)
```